recipes/xilinx-runtime/conda.patch

```diff
@@ -0,0 +1,223 @@
+ # target (say a static executable), use
+ #  set_target_properties(<target> PROPERTIES INSTALL_RPATH "")
+-SET(CMAKE_INSTALL_RPATH "$ORIGIN/../lib${LIB_SUFFFIX}:$ORIGIN/../..:$ORIGIN/../../lib${LIB_SUFFIX}")
++#SET(CMAKE_INSTALL_RPATH "$ORIGIN/../lib${LIB_SUFFFIX}:$ORIGIN/../..:$ORIGIN/../../lib${LIB_SUFFIX}")
+ 
+ # --- Release: eula ---
+ file(GLOB XRT_EULA
+@@ -180,7 +180,7 @@
+   PERMISSIONS OWNER_READ OWNER_EXECUTE OWNER_WRITE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
+   DESTINATION ${XRT_INSTALL_DIR}/test)
+ 
+-add_subdirectory("../tests/validate" "${CMAKE_CURRENT_BINARY_DIR}/validate_build")
++#add_subdirectory("../tests/validate" "${CMAKE_CURRENT_BINARY_DIR}/validate_build")
+ message("-- XRT version: ${XRT_VERSION_STRING}")
+ 
+ # -- CPack
+@@ -188,10 +188,10 @@
+ 
+ set (XRT_DKMS_DRIVER_SRC_BASE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/runtime_src/core")
+ 
+-include (CMake/dkms.cmake)
+-include (CMake/dkms-aws.cmake)
+-include (CMake/dkms-azure.cmake)
+-include (CMake/dkms-container.cmake)
++#include (CMake/dkms.cmake)
++#include (CMake/dkms-aws.cmake)
++#include (CMake/dkms-azure.cmake)
++#include (CMake/dkms-container.cmake)
+ 
+ # --- ICD ---
+ include (CMake/icd.cmake)
+diff -Naur XRT-202210.2.13.466/src/CMake/pkgconfig.cmake work/src/CMake/pkgconfig.cmake
+--- XRT-202210.2.13.466/src/CMake/pkgconfig.cmake	2022-04-14 12:29:02.000000000 -0700
++++ work/src/CMake/pkgconfig.cmake	2022-06-19 10:05:21.666383718 -0700
+@@ -11,6 +11,8 @@
+   set(XRT_PKG_CONFIG_DIR "/usr/share/pkgconfig")
+ endif ()
+ 
++set(XRT_PKG_CONFIG_DIR "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig")
++
+ configure_file (
+   ${CMAKE_SOURCE_DIR}/CMake/config/xrt.pc.in
+   xrt.pc
+diff -Naur XRT-202210.2.13.466/src/CMake/version.cmake work/src/CMake/version.cmake
+--- XRT-202210.2.13.466/src/CMake/version.cmake	2022-04-14 12:29:02.000000000 -0700
++++ work/src/CMake/version.cmake	2022-06-19 10:06:34.002814573 -0700
+@@ -63,8 +63,8 @@
+ endif()
+ 
+ # This is not required on MPSoC platform. To avoid yocto error, do NOT intall
+-if ((${XRT_NATIVE_BUILD} STREQUAL "yes") AND (NOT WIN32))
+-  # Copied over from dkms.cmake. TODO: cleanup
+-  set (XRT_DKMS_INSTALL_DIR "/usr/src/xrt-${XRT_VERSION_STRING}")
+-  install(FILES ${CMAKE_BINARY_DIR}/gen/version.h DESTINATION ${XRT_DKMS_INSTALL_DIR}/driver/include)
+-endif()
++#if ((${XRT_NATIVE_BUILD} STREQUAL "yes") AND (NOT WIN32))
++#  # Copied over from dkms.cmake. TODO: cleanup
++#  set (XRT_DKMS_INSTALL_DIR "/usr/src/xrt-${XRT_VERSION_STRING}")
++#  install(FILES ${CMAKE_BINARY_DIR}/gen/version.h DESTINATION ${XRT_DKMS_INSTALL_DIR}/driver/include)
++#endif()
+diff -Naur XRT-202210.2.13.466/src/CMakeLists.txt work/src/CMakeLists.txt
+--- XRT-202210.2.13.466/src/CMakeLists.txt	2022-04-14 12:29:02.000000000 -0700
++++ work/src/CMakeLists.txt	2022-06-19 10:07:59.471327068 -0700
+@@ -48,8 +48,10 @@
+   -Wno-overloaded-virtual
+    )
+ else()
+-  set(XRT_WARN_OPTS -Wall -Werror )
++  set(XRT_WARN_OPTS -Wall)
```

Comment:
@timsnyder yes this gets picked up in the build. CMake will not treat warnings as errors. XRT team was trying to force all warnings to be errors, so that developers would actually fix their warnings. However, this is too draconian when you consider that you need the world to compile your stuff in different environments. In the latest greatest XRT, there is now an option to turn this off, however here, I have to hack it in the patch.